librz/analysis/var_global.c

```diff
@@ -3,23 +3,38 @@
 
 #include <rz_analysis.h>
 #include <rz_list.h>
+#include <rz_core.h>
```

Comment:
This should not be done. RzCore module depends on RzAnalysis, RzBin, RzUtil and others. They can't/should not depend on RzCore.